test-suite/test-cases/rdfa1.1/xhtml1/0198.sparql

```diff
@@ -1,7 +1,7 @@
 ASK WHERE {
  <http://www.example.org/me#mark> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://xmlns.com/foaf/0.1/Person> .
  <http://www.example.org/me#mark> <http://xmlns.com/foaf/0.1/name>
- "<span property=\"foaf:firstName\" xmlns=\"http://www.w3.org/1999/xhtml\" xmlns:foaf=\"http://xmlns.com/foaf/0.1/\" xmlns:rdf=\"http://www.w3.org/1999/02/22-rdf-syntax-ns#\">Mark</span> <span property=\"foaf:surname\" xmlns=\"http://www.w3.org/1999/xhtml\" xmlns:foaf=\"http://xmlns.com/foaf/0.1/\" xmlns:rdf=\"http://www.w3.org/1999/02/22-rdf-syntax-ns#\">Birbeck</span>"^^<http://www.w3.org/1999/02/22-rdf-syntax-ns#XMLLiteral> .
+ "<span property=\"foaf:firstName\">Mark</span> <span property=\"foaf:surname\">Birbeck</span>"^^<http://www.w3.org/1999/02/22-rdf-syntax-ns#XMLLiteral> .
```

Comment:
The namespace declarations are necessary for XMLLiterals, as they are in-scope when the literal is extracted. This is because of[Exclusive XML Canonicalization](https://www.w3.org/TR/2006/REC-xml-20060816/), and the note in the RDFa spec indicates that the namespace declarations are necessary.

> In order to maintain maximum portability of this literal, any children of the current node that are elements must have the current XML namespace declarations (if any) declared on the serialized element. Since the child element node could also declare new XML namespaces, the RDFa Processor must be careful to merge these together when generating the serialized element definition. For avoidance of doubt, any re-declarations on the child node must take precedence over declarations that were active on the current node.